File: 1528-kids-with-the-greatest-number-of-candies/1528-kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        result = [False]*len(candies)
        max_a = max(candies)
        for i in range(len(candies)):
            if extraCandies+candies[i]>=max_a:
                result[i]= True

        return result

File: 1528-kids-with-the-greatest-number-of-candies/1528-kids-with-the-greatest-number-of-candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        vector<bool> result(candies.size(), false);
        for(int i =0; i<candies.size();i++){
            if(candies.at(i) + extraCandies >= *max_element(candies.begin(), candies.end())){
                result.at(i)=true;
            }
        }
        return result;

        
    }
};